main.go

```diff
@@ -1,42 +1,46 @@
 package main
 
 import (
+	"context"
 	"fmt"
 
 	pipelineDefaults "github.com/equinor/radix-operator/pipeline-runner/model/defaults"
 	"github.com/equinor/radix-tekton/pkg/kubernetes"
 	"github.com/equinor/radix-tekton/pkg/models"
 	"github.com/equinor/radix-tekton/pkg/models/env"
 	"github.com/equinor/radix-tekton/pkg/pipeline"
-	log "github.com/sirupsen/logrus"
+	"github.com/equinor/radix-tekton/pkg/utils/logger"
+	"github.com/rs/zerolog/log"
 )
 
 func main() {
 	environment := env.NewEnvironment()
-	setLogLevel(environment)
+	level, err := environment.GetLogLevel()
+	if err != nil {
+		log.Fatal().Err(err).Msg("Failed to initialize log level")
+	}
+	logger.InitializeLogger(context.Background(), level, true)
+	if err != nil {
```

Comment:
This `err` is not returned by InitializeLogger(), but comes from GetLogLevel()

Reply:
:O  Fixed